lib/utils/string-utils.ts

```diff
@@ -27,3 +27,23 @@ export function kebabToCamel(str: string) { return str.replace(/-./g, x => x[1].
  * @returns
  */
 export function escapeDots(str: string) { return str.replace(/\./g, '\\.'); }
+
+/**
+ * Removes text between given tokens or the just the tokens themselves.
+ * Example use case: YAML manipulation similar to Helm: openToken = "{{ if ... }}", closeToken = "{{ end }}""
+ * @param string
+ * @returns
+ */
+export function changeTextBetweenTokens(str: string, openToken: string, closeToken: string, keep: boolean) {
```

Comment:
i like this implementation. @shapirov103 Do we have to create a doc page for string utils?

Reply:
We have not done it before, the utils part is not the main value of the blueprints. I would just put more emphasis on the JSDoc and rely on the API doc tool for those for now.